mcodingbot/utils/search.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import annotations
+
+from typing import Mapping, TypeVar
+
+from rapidfuzz import fuzz, process
+
+K = TypeVar("K")
+V = TypeVar("V")
+
+
+def fuzzy_search(
```

Comment:
Removes default values being assigned in different places. They are already different so it's an issue.
```python
def fuzzy_search(
    query: str,
    choices: Mapping[K, V],
    *,
    score_cutoff: int | float | None,
    limit: int,
) -> list[tuple[V, float, K]]:
```

Reply:
can you make this into a suggestion pls?

Reply:
No. I have locked myself out of the room with my computer so I'm a bit fucked rn.